tools/gen-intrinsics-specs.py

```diff
@@ -787,9 +968,10 @@ def get_intrinsics_db(path):
                         help="CSV file that map the intrinsics to their classification.", required=True)
     parser.add_argument("--outfile", metavar="<path>", type=str,
                         help="Output file where the RST of the specs is written.", required=True)
+    parser.add_argument("--source_syntax", metavar="<path>", type=str,
+                        help="The type of syntax the output file should be rendered in. Can be rst or markdown.", required=True)
```

Comment:
If only two choices are available, we better use the ArgumentParser to enforce them: https://docs.python.org/3/library/argparse.html#choices